sor/src/main/java/com/bazaarvoice/emodb/sor/core/DataTools.java

```diff
@@ -40,6 +41,11 @@
     Iterator<MultiTableScanResult> multiTableScan(MultiTableScanOptions query, TableSet tables, LimitCounter limit, ReadConsistency consistency);
 
     /**
+     * For performing a raw scan across multiple tables with cut off time. No Changes after the given cut off time will be included.
+     */
+    Iterator<MultiTableScanResult> multiTableScanWithCutoffTime(MultiTableScanOptions query, TableSet tables, LimitCounter limit, ReadConsistency consistency, UUID time);
```

Comment:
Don't know if this needs to be a distinct call from the previous `multiTableScan()`  Maybe just a single method with a `@Nullable` time, where providing `null` returns current behavior.